x/ccv/provider/types/params.go

```diff
@@ -36,6 +36,14 @@ const (
 	// that is replenished to the slash meter every replenish period. This param also serves as a maximum
 	// fraction of total voting power that the slash meter can hold.
 	DefaultSlashMeterReplenishFraction = "0.05"
+
+	// DefaultBlocksPerEpoch defines the default blocks that constitute an epoch. Assuming we need 6 seconds per block,
+	// an epoch corresponds to 1 hour (6 * 600 = 3600 seconds).
+	DefaultBlocksPerEpoch = 600
+
+	// MaxBlocksPerEpoch defines the maximum blocks that constitute an epoch. Assuming we need 6 seconds per block,
+	// the maximum epoch corresponds to 2 hours (6 * 1200 = 7200 seconds).
+	MaxBlocksPerEpoch = 1200
```

Comment:
Don't do this

Reply:
Agreed. Why force someone to update this if e.g. in the future block times are decreased to 0.1s? Seems unnecessary to have